src/Storages/System/StorageSystemMoves.cpp

```diff
@@ -0,0 +1,47 @@
+#include <Interpreters/Context.h>
+#include <Storages/MergeTree/MovesList.h>
+#include <Storages/System/StorageSystemMoves.h>
+#include <Access/ContextAccess.h>
+
+
+namespace DB
+{
+
+NamesAndTypesList StorageSystemMoves::getNamesAndTypes()
+{
+    return {
+        {"database", std::make_shared<DataTypeString>()},
+        {"table", std::make_shared<DataTypeString>()},
+        {"elapsed", std::make_shared<DataTypeFloat64>()},
+        {"target_disk_name", std::make_shared<DataTypeString>()},
+        {"target_disk_path", std::make_shared<DataTypeString>()},
+        {"partition_id", std::make_shared<DataTypeString>()},
+        {"part_size", std::make_shared<DataTypeUInt64>()},
+        {"thread_id", std::make_shared<DataTypeUInt64>()},
```

Comment:
If I look right this thread_id is equal to the the thread id which has scheduled the move, not the one which is actually moving parts.
Is I right? Is it what expected?

Reply:
Hm, it's done 100% the same way as it is done for `system.merges` and `system.replicated_fetches`. Let me check maybe it is actually broken there also...

Reply:
Looks like it is real problem only for moves, because entry is created on "tagging", so it is called for different place for moves. I have to fix it.